en/docs/references/choreo-expression-syntax.md

```diff
@@ -0,0 +1,344 @@
+# Choreo Expression Syntax
+
+Choreo expressions comprise a powerful expression syntax based on the Ballerina language for function invocations, declaring a variable using literals and tuples, typecasting, etc. This document presents you the common expressions that you can use while developing Choreo applications.
```

Comment:
Should we link to ballerina.io from the text `Ballerina language` above?

Reply:
Yes, that seems better.